ImputationPipeline/AggregatePRSResults.wdl

```diff
@@ -198,7 +200,7 @@ task BuildHTMLReport {
 
     ## Control Sample
     \`\`\`{r control, echo = FALSE, results = "asis"}
-    kable(list(batch_control_results, expected_control_results) %>% reduce(bind_rows) %>% select(-batch_id) %>% select(-is_control) , digits = 2)
+    kable(list(batch_control_results, expected_control_results) %>% reduce(bind_rows) %>% select(-batch_id, -is_control) %>% add_column(Sample=c('Batch control', 'Expected control'), .before='sample_id'), digits = 2)
```

Comment:
I think we should drop the sample_id column as well here.  All we care about is which is the batch control, and which is the expected control, the sample id of the control is unimportant.